rector-ci.php

```diff
@@ -28,7 +29,14 @@
                 'Exception',
                 'Doctrine\ORM\EntityManagerInterface',
                 'Doctrine\ORM\EntityManager',
-            ]
+            ],
+        ]]);
+
+    $services->set(PreferThisOrSelfMethodCallRector::class)
+        ->call('configure', [[
+            PreferThisOrSelfMethodCallRector::TYPE_TO_PREFERENCE => [
+                'PHPUnit\Framework\TestCase' => 'self',
```

Comment:
```suggestion
                'PHPUnit\Framework\TestCase' => '$this',
```

Also, it should be added somehow to constant in `PreferThisOrSelfMethodCallRector`,

e.g. 

```php
PreferThisOrSelfMethodCallRector::TYPE_SELF
PreferThisOrSelfMethodCallRector::TYPE_THIS
PreferThisOrSelfMethodCallRector::TYPE_STATIC
```

Reply:
updated use `PreferThisOrSelfMethodCallRector::PREFER_THIS`